neat/tools/fittools/compartmentfitter.py

```diff
@@ -886,17 +986,38 @@ def plotKernels(self, alphas, phimat, t_arr=None):
                 pstring = '%d $\leftrightarrow$ %d'%(ii,jj)
                 ax.set_title(pstring, pad=-10)
 
-    def checkPassive(self, loc_arg, alpha_inds=[0], recompute=False, n_modes=5,
-                           use_all_chans_for_passive=True, force_tau_m_fit=False):
+    def checkPassive(self, loc_arg, alpha_inds=[0], n_modes=5,
+                           use_all_channels_for_passive=True, force_tau_m_fit=False,
+                           recompute=False, pprint=False):
+        """
+        Checks the impedance kernels of the passive model.
+
+        Parameters
+        ----------
+        loc_arg: list of locations or string (see documentation of
+                :func:`MorphTree._convertLocArgToLocs` for details)
+            The compartment locations
+        n_modes: int
+            The number of eigen modes that are shown
+        alpha_inds: list of ints
+            Indices of all mode time-scales to be included in the fit
```

Comment:
please keep the correct order of arguments also in the docstring

Reply:
done